cmd/service/assets/readmeTemplate.go

```diff
@@ -180,11 +180,12 @@ type bintree struct {
 	Func     func() (*asset, error)
 	Children map[string]*bintree
 }
+
 var _bintree = &bintree{nil, map[string]*bintree{
-	"cmd": &bintree{nil, map[string]*bintree{
-		"service": &bintree{nil, map[string]*bintree{
-			"assets": &bintree{nil, map[string]*bintree{
-				"readmeTemplate.md": &bintree{cmdServiceAssetsReadmetemplateMd, map[string]*bintree{}},
+	"cmd": {nil, map[string]*bintree{
```

Comment:
this is a generated file we shouldn't update it

Reply:
File re-generated